controllers/specialresource.go

```diff
@@ -49,12 +54,23 @@ import (
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=prometheusrules,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=route.openshift.io,resources=routes,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=config.openshift.io,resources=clusteroperators,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=config.openshift.io,resources=clusteroperators/status,verbs=get;list;watch;create;update;patch;delete
+
 // ReconcilerSpecialResources Takes care of all specialresources in the cluster
 func ReconcilerSpecialResources(r *SpecialResourceReconciler, req ctrl.Request) (ctrl.Result, error) {
 
-	r.Log.Info("Reconciling SpecialResource(s) in all Namespaces")
+	log = r.Log.WithName(prettyPrint("preamble", Purple))
 
+	log.Info("Reconcling ClusterOperator")
+	if err := r.clusterOperatorStatusReconcile(); err != nil {
+		log.Info("Reconcling ClusterOperator failed", "error", fmt.Sprintf("%v", err))
```

Comment:
Another nit, for later reference (don't need to hold the merge for this)
```suggestion
	log.Info("Reconciling ClusterOperator")
	if err := r.clusterOperatorStatusReconcile(); err != nil {
		log.Info("Reconciling ClusterOperator failed", "error", fmt.Sprintf("%v", err))
```